bazel/test.bzl

```diff
@@ -58,6 +58,7 @@ def _redpanda_cc_test(
         memory,
         cpu,
         srcs = [],
+        defines = [],
```

Comment:
would `local_defines` be more appropriate here? don't think it makes much difference since tests are most likely dependency tree leaves - more so considering "best practices" looking forward.

Reply:
Tests don't have the concept of `local_defines` only libraries do, exactly because they are leaves. Reference: be.bazel.build

Reply:
ya makes sense - i was looking at [this](https://bazel.build/reference/be/c-cpp#cc_test) and fixed on

>When in doubt, add define values to [local_defines](https://bazel.build/reference/be/c-cpp#cc_binary.local_defines) instead.

Though I imagine that's copy/pasted from a library rule or something.

Reply:
Oh there does seem to a local defines attribute... I have no idea how you could depend on a cc_test in a way that would need both...

Reply:
I think it's literal copy/pasta tbh... even that link there in the quoted text points to the attribute in `cc_binary` 🤷